en/debug/eclipse_jlink.md

```diff
@@ -78,6 +78,16 @@ For more information, see: [https://gnu-mcu-eclipse.github.io/debug/jlink/instal
    ![Eclipse: GDB Segger Debug config: debugger tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_debugger_tab.png)
    ![Eclipse: GDB Segger Debug config: startup tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_startup_tab.png)
 
+## SEGGER Task-aware debugging
+
+1. Make sure that in your build the following config is enabled *Enable TCBinfo struct for debug* `CONFIG_DEBUG_TCBINFO`
```

Comment:
What is `CONFIG_DEBUG_TCBINFO` that is sitting at the end of this line? It looks like a typo - but it could be this is a setting you configure somewhere they is also set using the menuconfig (i.e. this is the actual setting, and menuconfig is hides this behind "Enable TCBinfo struct for debug".

Reply:
It's the internal config symbol used by NuttX